src/components/cylc/TreeItem.vue

```diff
@@ -129,7 +141,7 @@ export default {
     getTypeStyle () {
       const styles = {}
       if (this.hasChildren) {
-        styles['cursor'] = 'pointer'
+        styles.cursor = 'pointer'
```

Comment:
Needed for the linter, happy to remove it if other PR fixes it first (I think I fixed it in the progress percent PR too)